ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -149,7 +149,7 @@ class ConfirmPaymentMethodScreen extends React.Component<Props, never> {
           <PaymentBannerComponent
             currentAmount={currentAmount}
             paymentReason={paymentReason}
-            recipient={recipient || UNKNOWN_RECIPIENT}
+            recipient={recipient || UNKNOWN_RECIPIENT_EMPTY}
```

Comment:
Can't we just make "recipient" prop optional and check it in PaymentBannerComponent?